### ISSUE TITLE
Drag lag after updating Chrome on MacOS

### ISSUE THREAD

**Issue by Konstantinov-Innokentii** [bokuweb/react-rnd#603]
**Is it a bug/feature/question or do you need help?**  bug/help
**If it's a bug, is it a browser bug?** Not sure

### Overview of the problem

I'm using react-rnd **10.1.0**   
      
My **browser** is:   Chrome 78 
   
I am sure this issue is **not a duplicate**? Yes
   

### Description

After updating Chrome to 78 I noticed, that content inside rnd-component "freeze" after drag. (Image bellow). I faced this issue only using Chrome 78 on MacOs. On Safari, Chrome on Windows, earlier Chrome versions on Mac all seems to be ok.
<img width="1552" alt="Снимок экрана 2019-11-06 в 22 09 24" src="https://user-images.githubusercontent.com/20221722/68321495-a1c77100-00e3-11ea-8ddd-1cea6e9d67fd.png">

Lot of thanks.

**Comment by ignapas** [comment]
I believe this is some problem with ``overflow: auto``, overflowing content, scroll bars... I also encountered and I am trying to solve it. It might be related to Chrome and not the library...

**Comment by yuriy-klerk** [comment]
> I believe this is some problem with `overflow: auto`, overflowing content, scroll bars... I also encountered and I am trying to solve it. It might be related to Chrome and not the library...

Yes, many thanks. I applied `overflow` property value as `auto` in the bounds container and then dragging functionality are works correctly.

**Comment by VictorNovice** [comment]
@yuriy-klerk Could you please describe your solution in more detail?

**Comment by Konstantinov-Innokentii** [comment]
My solution was:
`.react-draggable{
    overflow: auto;
}`

**Comment by IdanGoor** [comment]
I'm also dealing with this issue. @Konstantinov-Innokentii your solution indeed works (also when you set overflow:hidden), but unfortunately for me it conflicts with the design of the handles. Can you recall on which version of Chrome it worked fine for you?

**Comment by Konstantinov-Innokentii** [comment]
@IdanGoor on 78.0.3904.108, macOS

**Comment by yuriy-klerk** [comment]
@VictorNovice 
 ` <Rnd style={{
                                            "position": "relative",
                                            "zIndex": 1,
                                            "cursor": "move",
                                            **"overflow": "hidden"**
                                        }}
 >`

Try this solution.

**Comment by yuriy-klerk** [comment]
Also you can check drag and drop functionality based on React-Rnd in my project:
https://callaba.io/
Try create room and try to drag some partipiants
